ies-route53/recipes/add.rb

```diff
@@ -0,0 +1,22 @@
+require 'net/http'
+Chef::Resource.send(:include, EasyBib)
+
+include_recipe 'route53'
+
+route53_record 'create a record' do
+  host_name = get_hostname(node, true)
+  stack_name = get_normalized_cluster_name(node)
+  zone_name = node['ies-route53']['zone']['name']
+  name "#{host_name}.#{stack_name}.#{zone_name}"
+  value Net::HTTP.get(URI.parse('http://169.254.169.254/latest/meta-data/public-ipv4'))
```

Comment:
And this should be in a `node[]` attribute — `node['opsworks']['instance']['public_ip']` or something?